ipatests/test_integration/test_ipa_ipa_migration.py

```diff
@@ -1265,3 +1265,44 @@ def test_check_ad_idrange_is_migrated(self):
             ["ipa", "idrange-show", ad_domain_name + "_id_range"]
         )
         assert cmd1.stdout_text == cmd2.stdout_text
+
+
+class TestIPAMigratewithBackupRestore(IntegrationTest):
+    """
+    Tests for ipa-migrate tool with backup files
+    """
+    num_replicas = 1
```

Comment:
When a test class inherits from IntegrationTest and defines num_replicas = 1 and topology = "line", the setup creates a master and a replica.
That is not what you want to do here as you want to install the first node and then migrate its data to the 2nd node. 

You need to override the method install with something similar to the following:
```
@classmethod
    def install(cls, mh):
        tasks.install_master(cls.master, setup_dns=True, setup_kra=True)
        prepare_ipa_server(cls.master)
        tasks.install_master(cls.replicas[0], setup_dns=True, setup_kra=True,
                             extra_args=['--allow-zone-overlap'])
```